packages/contracts/test/voting/majority-voting.ts

```diff
@@ -37,58 +37,60 @@ describe('MajorityVotingMock', function () {
   function initializeMock(
     minParticipation: any,
     supportThreshold: any,
-    minDuration: any
+    minDuration: any,
+    minProposerVotingPower: any
   ) {
     return votingBase.initializeMock(
       dao.address,
       minParticipation,
       supportThreshold,
-      minDuration
+      minDuration,
+      minProposerVotingPower
     );
   }
 
   describe('initialize: ', async () => {
     it('reverts if trying to re-initialize', async () => {
-      await initializeMock(1, 2, 3);
+      await initializeMock(1, 2, 3, 0);
 
-      await expect(initializeMock(1, 2, 3)).to.be.revertedWith(
+      await expect(initializeMock(1, 2, 3, 0)).to.be.revertedWith(
         ERRORS.ALREADY_INITIALIZED
       );
     });
 
     it('reverts if min duration is 0', async () => {
-      await expect(initializeMock(1, 2, 0)).to.be.revertedWith(
+      await expect(initializeMock(1, 2, 0, 0)).to.be.revertedWith(
         customError('VoteDurationZero')
       );
     });
   });
 
-  describe('changeVoteConfig: ', async () => {
+  describe('changeVoteSettings: ', async () => {
```

Comment:
Probably out of the scope, but the naming on the SDK looked like `setPluginConfiguration` or something similar.

Reply:
For `TokenVotingSetup`, we also have `MintSettings` and `TokenSettings`. Would `PluginSettings` be ok?

Reply:
No problem. The main point is about using `setPluginXXX()`, not just token or minting